env.go

```diff
@@ -0,0 +1,95 @@
+package envconf
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"io"
+	"os"
+	"strings"
+)
+
+var (
+	ErrInvalidPair = errors.New("invalid pair for env variable")
+)
+
+const (
+	commentSymbol = '#'
+)
+
+type EnvConfig struct {
+	envs map[string]string
+}
+
+func NewEnvConf() *EnvConfig {
+	return &EnvConfig{envs: map[string]string{}}
+}
+
+func (e *EnvConfig) Parse(data io.Reader) error {
+	b := bufio.NewReader(data)
+	for {
+		n, err := b.ReadBytes('\n')
+		if err != nil {
+			if err != io.EOF {
+				return err
+			}
+			if len(n) == 0 {
+				return nil
+			}
+		}
+		n = bytes.Trim(n, " ")
+		n = bytes.Trim(n, "\n")
+		n = bytes.Trim(n, "\t")
+		if n[0] == commentSymbol {
+			continue
+		}
+		i := bytes.Index(n, []byte("="))
+		if i == -1 {
+			return ErrInvalidPair
+		}
+		key := string(n[:i])
```

Comment:
1. key and value also should be trimmed on white spaces
2. strings.Trim(value, "`\"")

Reply:
@antonmashko 
1. We must enable the user to specify a space at the beginning/end of the line/key;
2. ``` this is invalid chapter for `env` variables.

Reply:
1. What do you mean? If you try in bash define variable like `FOO="                   BAR"` and print `FOO` variable, result will be `BAR` without spaces.
2. Sorry, my fault. I meant `'` character instead of `